### PR TITLE
Refactor advertisers

### PR DESCRIPTION
**Pull request by dwick**
:eyeglasses: @madbook 

Monkey patching the login to add required info seemed like the least bad option because of the way the login templates are rendered on the server.

### CODE REVIEW

**Comment by dwick** [comment]
screen shot 2015-08-07 at 12 47 50 pm

File: reddit_about/public/static/js/advertising/advertising.js [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
+
+    $form.append('<input type="hidden" name="sponsor" value="true">');
+
+    $email
+      .attr('placeholder', label)
+      .data('validate-url', '/api/check_email.json?sponsor=true')
+      .data('validate-noclear', true);
+    $email.closest('.c-form-group')
+      .find('label')
+      .html(label);
```
Comment (JordanMilne) [inline reddit_about/public/static/js/advertising/advertising.js]: Should this be `.text()`? `label` comes from the translators

Comment (dwick) [inline reddit_about/public/static/js/advertising/advertising.js]: yes, thank you.

**Comment by madbook** [comment]
code looks fine, despite the monkey patching :P

left a question on the other PR that also relates to this one

:fish:

**Pull request merged**